src/components/FloatingAnchor/FloatingAnchor.module.less

```diff
@@ -0,0 +1,17 @@
+.floatingAnchor {
+    position: relative;
+
+    .anchor {
+        float: left;
+        margin-left: -20px;
+
+        SVG {
```

Comment:
Is there a reason this selector is targeting the all uppercase element name and not the all lowercase element name in the `FloatingAnchor.js` file? Can we just add a class name here instead of targeting the element?

Reply:
I'll add a class